api/Node.json

```diff
@@ -1207,95 +1207,120 @@
-              },
-              {
-                "version_added": "14",
-                "version_removed": "29",
-                "partial_implementation": true,
-                "notes": "<code>deep</code> defaults to <code>true</code>."
-              },
-              {
-                "version_added": true,
-                "version_removed": "14",
-                "partial_implementation": true,
-                "notes": "<code>deep</code> is a required parameter."
-              }
-            ],
+            "firefox": {
+              "version_added": true
+            },
+            "firefox_android": {
+              "version_added": true
+            },
             "ie": {
               "version_added": true
             },
-            "opera": [
-              {
-                "version_added": true
-              },
-              {
-                "version_added": null,
-                "notes": "<code>deep</code> as an optional parameter"
-              }
-            ],
-            "opera_android": [
-              {
-                "version_added": true
-              },
-              {
-                "version_added": null,
-                "notes": "<code>deep</code> as an optional parameter"
-              }
-            ],
+            "opera": {
+              "version_added": true
+            },
+            "opera_android": {
+              "version_added": true
+            },
             "safari": {
-              "version_added": true,
-              "notes": "<code>deep</code> optional parameter, default value is <code>false</code>"
+              "version_added": true
             },
             "safari_ios": {
-              "version_added": true,
-              "notes": "<code>deep</code> optional parameter, default value is <code>false</code>"
+              "version_added": true
+            },
+            "webview_android": {
+              "version_added": true
             }
           },
           "status": {
             "experimental": false,
             "standard_track": true,
             "deprecated": false
           }
+        },
+        "deep_defaults_to_false": {
+          "__compat": {
+            "description": "<code>deep</code> defaults to <code>false</code>",
+            "support": {
+              "chrome": {
+                "version_added": true
+              },
+              "chrome_android": {
+                "version_added": true
+              },
+              "edge": {
+                "version_added": true
+              },
+              "edge_mobile": {
+                "version_added": true
+              },
+              "firefox": [
+                {
+                  "version_added": "29"
+                },
+                {
+                  "version_added": "13",
+                  "version_removed": "29",
+                  "notes": "<code>deep</code> defaults to <code>true</code>."
```

Comment:
This should also have `"partial_implementation": true`

Reply:
OK, since I was the one who originally advised against "`partial_implementation`" earlier in this PR, I feel like I should say that yes, I agree with restoring them now that I see how the deep subfeature has been written. I was originally imagining a general deep parameter feature, but I think the way this is structured now is better.